packages/adapters/bitcoin/src/connectors/OKXConnector.ts

```diff
@@ -52,10 +52,15 @@ export class OKXConnector extends ProviderEventEmitter implements BitcoinConnect
   public async getAccountAddresses(): Promise<BitcoinConnector.AccountAddress[]> {
     const accounts = await this.wallet.getAccounts()
 
-    return accounts.map(account => ({
-      address: account,
-      purpose: 'payment'
-    }))
+    const accountList = await Promise.all(
+      accounts.map(async account => ({
+        address: account,
+        purpose: 'payment' as const,
+        publicKey: await this.wallet.getPublicKey()
```

Comment:
no path here?

Reply:
Let me validate that with Felipe - I'm not quite sure what would the devs need or if they need `path` tbh. I'll take a look

Reply:
`path` would be the derivaion path used to get this account, should be exposed by wallet (if exposed)